llvm/test/CodeGen/AMDGPU/GlobalISel/llvm.amdgcn.raw.buffer.load.i8u8i16u16.ll

```diff
@@ -0,0 +1,194 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 6
```

Comment:
Test name doesn't follow the file naming convention, but what does it add over the existing raw.buffer.load case? Just add the missed cases there? These are also the legacy non-pointer intrinsics